Update set-up-sso-microsoft-entra-id.md (#5925)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-sso-microsoft-entra-id.md

| Field | Value |
| ----- | ----- |
| **Log&nbsp;in&nbsp;with** | Microsoft Entra ID Single Tenant |
| **Client&nbsp;ID** | Paste the **Application (client) ID** recorded in the steps above |

> | **Client&nbsp;Secret** | Paste the **Client Secret** (remember to use the Secret Value instead of the Secret ID) recorded in the steps above; <br />**Note:** When the client secret expires, an Entra ID admin will have to generate a new one to be pasted into dbt Cloud for uninterrupted application access. |

Reviewer: ```suggestion
| **Client&nbsp;Secret** | Paste the **Client Secret** (remember to use the Secret Value instead of the Secret ID) from the steps above; <br />**Note:** When the client secret expires, an Entra ID admin will have to generate a new one to be pasted into dbt Cloud for uninterrupted application access. |
```
